build-test-batch.xml

```diff
@@ -4379,6 +4379,10 @@ log.sanitizer.enabled=false</echo>
 		/>
 	</target>
 
+	<target name="functional-smoke-firefox-jdk8">
+		<run-functional-test app.server.type="tomcat" database.type="mysql" database.version="5.7" test.portal.log.assert="true" />
+	</target>
```

Comment:
I would suggest trying to override the `browser.type` property here, before the `run-functional-test`.